app/views/layouts/application.html.erb

```diff
@@ -8,7 +8,14 @@
 
 </head>
 <body>
-  <%= yield %>
```

Comment:
If you remove this, there will be no place for your actual view file to go. I.e. anything you put in a `show.html.erb`, `edit.html.erb`, `index.html.erb` etc, will never show up because it has no place to insert itself.